Zotero/Controllers/Attachment Downloader/AttachmentDownloader.swift

```diff
@@ -848,11 +861,17 @@ extension AttachmentDownloader: URLSessionDownloadDelegate {
             }
         }
 
-        func checkFileResponse(for file: File, fileStorage: FileStorage) -> Swift.Error? {
+        func checkFileResponse(for file: File, fileStorage: FileStorage, downloadTask: URLSessionDownloadTask) -> Swift.Error? {
             let size = fileStorage.size(of: file)
-            if size == 0 || (size == 9 && (try? fileStorage.read(file)).flatMap({ String(data: $0, encoding: .utf8) }) == "Not found") {
+            if size == 0 || (size == 9 && (try? fileStorage.read(file)).flatMap({ String(data: $0, encoding: .utf8) })?.caseInsensitiveCompare("Not found") == .orderedSame) {
```

Comment:
And while you're fixing the code above, you can extract this bit into a standalone function and use it both here and in `RemoteAttachmentDownloadOperation`

Reply:
Check https://github.com/zotero/zotero-ios/commit/919a1d40e913e3b4182d0bc76cc0bbf4bab870f6 (commited directly)